store/src/java/com/zimbra/cs/account/Account.java

```diff
@@ -68,7 +68,10 @@ public void modify(Map<String, Object> attrs) throws ServiceException {
     }
 
     public void addAlias(String alias) throws ServiceException {
-        getProvisioning().addAlias(this, alias);
+        getProvisioning().addAlias(this, alias, false);
```

Comment:
prefer to call the overloading function if possible. Others as well.
```suggestion
        this.addAlias(this, alias, false);
```

Reply:
Done